docs/tutorial/part-one/index.md

```diff
@@ -198,191 +198,190 @@ export default () => (
+6. Om te benadrukken hoe dit je componenten herbruikbaar maakt, voeg nu een extra component `<Header />` toe aan de over-pagina, door volgende code toe te voegen aan het bestand `src/pages/over.js` op te slaan.
 
-```jsx:title=src/pages/about.js
+```jsx:title=src/pages/over.js
 import React from "react"
-import Header from "../components/header"
+import Header from "../components/header";
 
 export default () => (
   <div style={{ color: `teal` }}>
-    <Header headerText="About Gatsby" />
-    <Header headerText="It's pretty cool" /> {/* highlight-line */}
+    <Header headerText="Over Gatsby" />
+    <Header headerText="Dit is best wel cool" /> {/* highlight-line */}
     <p>Such wow. Very React.</p>
   </div>
 )
 ```
 
-![Duplicate header to show reusability](08-duplicate-header.png)
+![Dubbele kop om herbruikbaarheid te tonen](08-duplicate-header.png)
 
-And there you have it; A second header — without rewriting any code — by passing different data using props.
+En voila; Een tweede kop - zonder code te herschrijven - door verschillende gegevens door te geven met behulp van props.
 
-### Using layout components
+### Layout componenten gebruiken
 
-Layout components are for sections of a site that you want to share across multiple pages. For example, Gatsby sites will commonly have a layout component with a shared header and footer. Other common things to add to layouts include a sidebar and/or a navigation menu.
+Layout componenten zijn voor delen van een site die je over meerdere pagina's wilt delen. Gatsby-sites hebben bijvoorbeeld meestal een layout component met een gedeelde kop- en voettekst. Andere veel voorkomende dingen die je aan layouts kunt toevoegen, zijn een sidebar en/of een navigatiemenu.
 
-You’ll explore layout components in [**part three**](/tutorial/part-three/).
+Je gaat layoutcomponenten verkennen in [**deel drie**](/tutorial/part-three/).
 
-## Linking between pages
+## Linken tussen pagina's
 
-You'll often want to link between pages — Let's look at routing in a Gatsby site.
+Je zult vaak een koppeling willen maken tussen pagina's - Laten we eens kijken naar routing op een Gatsby-site.
 
-### ✋ Using the `<Link />` component
+### ✋ Gebruik van het `<Link />` component
 
-1.  Open the index page component (`src/pages/index.js`), import the `<Link />` component from Gatsby, add a `<Link />` component above the header, and give it a `to` property with the value of `"/contact/"` for the pathname:
+1. Open het indexpagina component (`src/pages/index.js`), importeer het `<Link />` component uit Gatsby, voeg een `<Link />` component toe boven de kop en geef het een `to` attribuut met de waarde van `"/contact/"` als padnaam:
 
 ```jsx:title=src/pages/index.js
 import React from "react"
-import { Link } from "gatsby" // highlight-line
-import Header from "../components/header"
+import { Link } from "gatsby"; // highlight-line
+import Header from "../components/header";
```

Comment:
I just noticed but looks like you need to set your "semi-colon" setting to not insert them, because I don't think it makes sense to add different code formatting as part of the language translation process :)